src/frontend/components/StudentViewersList/index.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import { Grommet } from 'grommet';
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+
+import { useParticipantsStore } from 'data/stores/useParticipantsStore';
+import { imageSnapshot } from 'utils/tests/imageSnapshot';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { theme } from 'utils/theme/theme';
+import { StudentViewersList } from '.';
+
+describe('<StudentViewersList />', () => {
+  it('adds and removes several users from the list.', () => {
+    render(wrapInIntlProvider(<StudentViewersList />));
+    expect(screen.queryByText('On stage')).toEqual(null);
+    screen.getByText('Other participants');
+
+    act(() =>
+      useParticipantsStore.getState().addParticipant({
+        isInstructor: true,
+        isOnStage: true,
+        name: 'Instructor',
+      }),
+    );
+    screen.getByText('Instructor');
+
+    act(() =>
+      useParticipantsStore.getState().addParticipant({
+        isInstructor: false,
+        isOnStage: false,
+        name: 'Student 1',
+      }),
+    );
+    screen.getByText('Student 1');
+
+    act(() =>
+      useParticipantsStore.getState().addParticipant({
+        isInstructor: false,
+        isOnStage: false,
+        name: 'Student 2',
+      }),
+    );
+    screen.getByText('Student 2');
+
+    act(() => useParticipantsStore.getState().removeParticipant('Student 2'));
+    expect(screen.queryByText('Student 2')).toEqual(null);
```

Comment:
```suggestion
    expect(screen.queryByText('Student 2')).not.toBeInTheDocument();
```

Reply:
done in [e02c29134a3b2b55b074b7787123df0695ddc6df](https://github.com/openfun/marsha/pull/1282/commits/e02c29134a3b2b55b074b7787123df0695ddc6df)